assets/src/dashboard/parts/components/Modal.js

```diff
@@ -19,7 +19,7 @@ export default function Modal({	icon, labels = {}, onRequestClose = () => {}, on
 		{
 			'bg-mango-yellow': 'warning' === variant
 		},
-		'optml__button flex justify-center px-5 py-3 rounded font-bold min-h-40 basis-1/5'
+		'inline-flex optml__button flex justify-center px-5 py-3 rounded font-bold min-h-40 basis-1/5'
```

Comment:
Why do we need to have this button as inline flex? It's already overridden later by the `flex` class.

Reply:
I've removed it.